amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/HierarchicalShardSyncerTest.java

```diff
@@ -174,59 +178,6 @@ public void testDetermineNewLeasesToCreate0Leases0ReshardsForMultiStream() {
         assertThat(newLeaseKeys, equalTo(expectedLeaseIds));
     }
 
-    /**
-     * Test determineNewLeasesToCreate() where there are no leases and no resharding operations have been performed, but
-     * one of the shards was marked as inconsistent.
-     */
-    @Test
-    public void testDetermineNewLeasesToCreate0Leases0Reshards1Inconsistent() {
```

Comment:
Why are the test cases removed? We should be refactoring it. Please make sure all the removed test cases are added and refactored.

Reply:
Removed these because for the empty lease table case we don't need to worry about inconsistent shards

Reply:
Re-added these tests, will see if we need them.

Need to add more unit tests in next revision